src/Arcus.Scripting.IntegrationAccount/Arcus.Scripting.IntegrationAccount.psm1

```diff
@@ -124,4 +124,56 @@ function Set-AzIntegrationAccountAssemblies {
     . $PSScriptRoot\Scripts\Set-AzIntegrationAccountAssemblies.ps1 -ResourceGroupName $ResourceGroupName -Name $Name -AssemblyFilePath $AssemblyFilePath -AssembliesFolder $AssembliesFolder -ArtifactsPrefix $ArtifactsPrefix
 }
 
-Export-ModuleMember -Function Set-AzIntegrationAccountAssemblies
\ No newline at end of file
+Export-ModuleMember -Function Set-AzIntegrationAccountAssemblies
+
+<#
+ .Synopsis
+  Upload/update a single, or multiple certificates into an Azure Integration Account.
+ 
+ .Description
+  Provide a file- or folder-path to upload/update a single or multiple certificates into an Integration Account.
+
+ .Parameter ResourceGroupName
+  The name of the Azure resource group where the Azure Integration Account is located.
+ 
+ .Parameter Name
+  The name of the Azure Integration Account into which the certificates are to be uploaded/updated.
+
+ .Parameter CertificateType
+  The type of certificate, this can either be Public or Private.
+
+ .Parameter CertificateFilePath
+  The full path of a certificate that should be uploaded/updated.
+  
+ .Parameter CertificatesFolder
+  The path to a directory containing all certificates that should be uploaded/updated.
+
+ .Parameter KeyName
+  The name of the key in Azure KeyVault that will be used for uploading/updating private certificates.
+
+ .Parameter KeyVersion
+  The version of the key in Azure KeyVault that will be used for uploading/updating private certificates.
+
+ .Parameter KeyVaultId
+  The id of the Azure KeyVault that will be used for uploading/updating private certificates.
+
+ .Parameter ArtifactsPrefix
+  The prefix, if any, that should be added to the certificates before uploading/updating.
+#>
+function Set-AzIntegrationAccountCertificates {
```

Comment:
I think this function is missing a comma in the `.psd1` file.

Reply:
Correct, fixed it in [`0f9065c` (#259)](https://github.com/arcus-azure/arcus.scripting/pull/259/commits/0f9065c869bcc322ecb87f747722b7e4a9647704)